statistics.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+
+
+def count_messages(chat):
+    count_messages_by_sender = dict()
+    for message in chat.messages:
+        user = message.sender
+        if user in count_messages_by_sender:
+            count_messages_by_sender[user] += 1
+        else:
+            count_messages_by_sender[user] = 1
+
+    return count_messages_by_sender
+
+
+def count_len_of_messages(chat):
+    count_len_by_sender = dict()
```

Comment:
Это было бы хорошим названием функции. Название переменной (если она не хранит в себе функцию конечно) не должно быть в виде "сделать что-нибудь"
```suggestion
    total_messages_length_by_sender = dict()
```